compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.h

```diff
@@ -121,8 +121,12 @@ std::unique_ptr<Pass> createAMDAIEControlCodeForallToForPass();
 /// Pass to unroll the loops within the control code regions.
 std::unique_ptr<Pass> createAMDAIEControlCodeLoopUnrollPass();
 
-/// Pass to convert control code DMA operations into NPU writes and syncs.
-std::unique_ptr<Pass> createAMDAIEControlCodeLoweringPass();
+/// Pass to convert control code DMA operations into HalfDmaCpyNd
+std::unique_ptr<Pass> createAMDAIEControlCodeToHalfDmaCpyNdPass();
+
+/// Pass to convert control code HalfDmaCpyNd into NPU WriteBd, AddressPatch,
+/// PushToQueue operations
+std::unique_ptr<Pass> createAMDAIEControlCodeToNpuPass();
```

Comment:
Could we keep the name the same (`createAMDAIEControlCodeLoweringPass`)? `createAMDAIEControlCodeToNpuPass` doesn't make sense to me because all these operations are already `npu` operations.